app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_zero_age() -> None:
+    assert get_human_age(0, 0) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(0, 0)}"
+    assert get_human_age(0, 10) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(0, 10)}"
+    assert get_human_age(10, 0) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(10, 0)}"
+
+
+def test_first_year_age() -> None:
+    assert get_human_age(1, 1) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(1, 1)}"
+    assert get_human_age(14, 14) == [0, 0],\
+        f"Expected [0, 0], but got {get_human_age(14, 14)}"
+    assert get_human_age(15, 15) == [1, 1],\
+        f"Expected [1, 1], but got {get_human_age(15, 15)}"
+
+
+def test_second_year_age() -> None:
+    assert get_human_age(16, 16) == [1, 1],\
+        f"Expected [1, 1], but got {get_human_age(16, 16)}"
+    assert get_human_age(23, 23) == [1, 1],\
+        f"Expected [1, 1], but got {get_human_age(23, 23)}"
+    assert get_human_age(24, 24) == [2, 2],\
+        f"Expected [2, 2], but got {get_human_age(24, 24)}"
```

Comment:
In `test_second_year_age`, the first two assertions have the same expected output [1, 1]. Consider if both are necessary or if one can be removed to avoid repetition.